src/system/game-data.ts

```diff
@@ -1170,8 +1170,8 @@ export class GameData {
 
   public async getRunHistoryData(scene: BattleScene): Promise<Object> {
     if (!Utils.isLocal) {
-      const data = await Utils.apiFetch("savedata/runHistory", true).json();
-      //const data = await response.json();
+      const dataAPI = await Utils.apiFetch("savedata/runHistory", true);
+      const data = await dataAPI.json();
```

Comment:
```suggestion
      const response = await Utils.apiFetch("savedata/runHistory", true);
      const data = await response.json();
```

you are dealing with responses, not some API